app/config/Module.scala

```diff
@@ -28,6 +29,7 @@ class Module extends AbstractModule {
     bind(classOf[DataRetrievalAction]).to(classOf[DataRetrievalActionImpl]).asEagerSingleton()
     bind(classOf[DataRequiredAction]).to(classOf[DataRequiredActionImpl]).asEagerSingleton()
     bind(classOf[PlaybackRepository]).to(classOf[PlaybackRepositoryImpl]).asEagerSingleton()
+    bind(classOf[Navigator]).to(classOf[OtherIndividualNavigator]).asEagerSingleton()
```

Comment:
This isn't strictly necessary as there is only one navigator needed in this service. You could copy everything over from OtherIndividualNavigator into Navigator